modules/bibupload/lib/bibupload.py

```diff
@@ -23,7 +23,7 @@
 tables according to options.
 """
 
-__revision__ = "$Id$"
+__revision__ = "6a4b1d3"
```

Comment:
Considering Invenio 1 is now rotting code, I don't think there is a need for this :smile: but what you did doesn't hurt.